services/localazy/localazy-all-progress.service.js

```diff
@@ -0,0 +1,32 @@
+'use strict'
+
+const LocalazyBase = require('./localazy-base.service')
+
+module.exports = class LocalazyAllProgress extends LocalazyBase {
+  static route = this.buildRoute()
+
+  static examples = [
+    {
+      title: 'Localazy translation progress',
+      keywords: this.keywords,
+      documentation: this.documentation,
+      namedParams: {
+        projectId: 'floating-apps',
+      },
+      staticPreview: this.render({
+        message: '61%',
+        label: 'translated',
+        color: '#066fef',
+      }),
+    },
+  ]
+
+  async handle({ projectId }) {
+    const { message, label, color } = await this.fetch({
```

Comment:
Agreed. If there's a need to use a custom scale that'd probably be okay, but the colors used for those tiers should follow our conventions to be consistent with our offering.

Reply:
We return colour to quickly generate the badges on our side, though in this PR, it's always possible to override it as well as the other parameters.

Reply:
What is the convention here? The gradient would certainly make sense for single language progress, even though it's not that simple as some code coverage for example. The rest was supposed to be just an informational value, since 100% on the project doesn't mean that the localization process is necessarily finished.